code/modules/modular_computers/computers/item/computer_ui.dm

```diff
@@ -59,21 +59,22 @@
 	data["device_theme"] = device_theme
 	data["login"] = list()
 
-	if(computer_id_slot)
-		var/stored_name = saved_identification
-		var/stored_title = saved_job
-		if(!stored_name)
-			stored_name = "Unknown"
-		if(!stored_title)
-			stored_title = "Unknown"
-		data["login"] = list(
-			IDName = saved_identification,
-			IDJob = saved_job,
-		)
-		data["proposed_login"] = list(
-			IDName = computer_id_slot.registered_name,
-			IDJob = computer_id_slot.assignment,
-		)
+	var/stored_name = saved_identification
+	var/stored_title = saved_job
+	if(!stored_name)
+		stored_name = "Unknown"
+	if(!stored_title)
+		stored_title = "Unknown"
+	data["login"] = list(
+		IDName = saved_identification,
+		IDJob = saved_job,
+	)
+
+	data["proposed_login"] = computer_id_slot ? list(
+		IDName = computer_id_slot.registered_name,
+		IDJob = computer_id_slot.assignment,
+	) : list()
```

Comment:
I think this part is better (for readability) how it was before.

Reply:
It wasn't working before.